src/ElementUtilities.js

```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```

Comment:
Also, regardless: add a test for the case of whether the element itself can / is expected to be returned when it matches.